prime-angular-frontend/src/app/modules/site-registration/pages/site-management-page/site-management-page.component.scss

```diff
@@ -64,3 +66,8 @@ div.add-site {
 .green-icon {
   color: theme-palette(green);
 }
+
+.org_agr_link {
```

Comment:
I think spell it out more, like `org_agreement_link`

Reply:
The name is updated.